pipeline/experiment.py

```diff
@@ -277,6 +310,63 @@ class ActionEvent(dj.Imported):
     action_event_time : decimal(8,4)  # (s) from trial start
     """
 
+
+# ---- Foraging paradigm specifics ----
+
+@schema
+class SessionBlock(dj.Imported):
+    definition = """
+    -> Session
+    block : smallint 		# block number
+    ---
+    block_uid : int  # unique across sessions/animals 
```

Comment:
I have added this to make it similar to the trial table.
I find this block_uid and trial_uid useless, datajoint is much more powerful than this workaround. I can't think of a use of these uid-s that couldn't be done only with datajoint. 
Another thing is that I am ingesting the subjects parallel, so in my script I can't even populate this uid entry properly.
I suggest leaving it out.

Reply:
I agree, we have not really make use of the `trial_uid` at all, as you said, DataJoint can always uniquely identify a trial or block based on the primary key. I'd suggest leaving it out as well.